includes/class-wporg-notifications.php

```diff
@@ -89,40 +89,16 @@ function () {
 	 */
 	public static function get_validator_email_addresses( WP_Comment $comment, array $comment_meta ): array {
 		$locale                 = $comment_meta['locale'][0];
-		$emails                 = self::get_gte_email_addresses( $locale );
-		$emails                 = array_merge( $emails, self::get_pte_email_addresses_by_project_and_locale( $comment, $locale ) );
-		$emails                 = array_merge( $emails, self::get_clpte_email_addresses_by_project( $comment ) );
-		$parent_comments        = self::get_parent_comments( $comment->comment_parent );
-		$emails_from_the_thread = self::get_commenters_email_addresses( $parent_comments, '' );
-		// Set the emails array as empty if one GTE/PTE/CLPTE has a comment in the thread.
-		if ( true !== empty( array_intersect( $emails, $emails_from_the_thread ) ) || ( in_array( $comment->comment_author_email, $emails ) ) ) {
-			$emails = array();
-		}
-		return $emails;
-	}
-
-	/**
-	 * Gets the emails to be notified from the thread comments.
-	 *
-	 * Removes the second parameter from the returned array if it is found.
-	 *
-	 * @since 0.0.2
-	 *
-	 * @param array  $comments                Array with the parent comments to the posted comment.
-	 * @param string $email_address_to_remove Email from the posted comment.
-	 *
-	 * @return array The emails to be notified in the thread.
-	 */
-	public static function get_commenters_email_addresses( array $comments, string $email_address_to_remove ): array {
```

Comment:
@amieiro I've removed this since it's the same implementation as in `GP_Notifications`, no need to repeat this here.